views/pages/players.html

```diff
@@ -12,4 +12,95 @@
   </tr>
   {{end}}
 </table>
+<div class="flex min-h-full flex-col justify-center px-6 py-12 lg:px-8">
+  <div class="sm:mx-auto sm:w-full sm:max-w-sm">
+    <img class="mx-auto h-10 w-auto" src="/public/icons/soccer-football-svgrepo-com.svg" alt="Scouting Pro">
+    <h2 class="mt-10 text-center text-2xl font-bold leading-9 tracking-tight text-gray-900">Add a player</h2>
+  </div>
+
+  <div class="mt-10 sm:mx-auto sm:w-full sm:max-w-sm">
+    <form class="space-y-6" hx-post="/players" hx-target="#result" hx-swap="innerHTML">
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="name" class="block text-sm font-medium leading-6 text-gray-900">Full Name:</label>
+          <div class="mt-2">
+            <input type="text" id="name" name="name" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="birthdate" class="block text-sm font-medium leading-6 text-gray-900">Birthdate (yyyy-mm-dd):</label>
```

Comment:
From the screenshot, this says `yyyy-mm-dd` but the input field is `mm/dd/yyyy`

Reply:
We should probably use whatever users would be most familiar with, even if we store it as yyyy-mm-dd in the database.

Reply:
I couldn't change the default input format, that stays as `mm/dd/yyyy` and there is no other way beside using js library so lets stick with default